src/pkg/message/credentials.go

```diff
@@ -22,61 +22,19 @@ const (
 	AgentKey        = "agent"
 )
 
-// PrintCredentialTable displays credentials in a table
-func PrintCredentialTable(state *types.ZarfState, componentsToDeploy []types.DeployedComponent) {
-	if len(componentsToDeploy) == 0 {
-		componentsToDeploy = []types.DeployedComponent{{Name: "git-server"}}
-	}
-
-	// Pause the logfile's output to avoid credentials being printed to the log file
-	if logFile != nil {
-		logFile.Pause()
-		defer logFile.Resume()
-	}
```

Comment:
this was not ported over, but logFile is only used with the legacy logger, and since this now goes to stdout it doesn't go to the logfile anyway